website/js/VariantTable.js

```diff
@@ -95,24 +97,62 @@ const researchModeGroups = [
+    // the reportBinding field must also be specified, and describes how individual reports are bound to nested subtiles
+    // (note that specific sources are hardcoded to have different appearances in components/VariantSubmitter.js)
+    {groupTitle: 'Clinical Significance (ClinVar)', internalGroupName: 'Significance (ClinVar)', reportSource: 'ClinVar',
+        reportBinding: {
+            sortBy: (a, b) => {
+                // sort first by stars, then by date in reverse chronological order
+                // (we receive strings for the date, so we have to first interpret them as dates)
+                // FIXME: is there a more reliable way to parse these strings as dates?
+
+                const starDiff = (
+                    computeReviewStatusScore(b['Review_Status_ClinVar']) -
+                    computeReviewStatusScore(a['Review_Status_ClinVar'])
+                );
+
+                const datetimeDiff = (
+                    new Date(b['Date_Last_Updated_ClinVar']).getTime() -
+                    new Date(a['Date_Last_Updated_ClinVar']).getTime()
+                );
+
+                return starDiff || datetimeDiff;
+            },
+            submitter: {title: 'Submitter', prop: 'Submitter_ClinVar'},
+            cols: [
+                // displayed here in full since the display in the header is potentially truncated
+                {title: 'Submitter', prop: 'Submitter_ClinVar'},
+
+                // added these fields (redundantly?) b/c they're not noticeable in the header
+                {title: 'Clinical Significance', prop: 'Clinical_Significance_ClinVar'},
+                {title: 'Date Last Updated', prop: 'Date_Last_Updated_ClinVar'},
+
+                {title: 'Submission Type', prop: 'Method_ClinVar'}, // likely Method_ClinVar
+                {title: 'SCV Accession', prop: 'SCV_ClinVar'},
+                {title: 'Summary Evidence', prop: 'Summary_Evidence_ClinVar'},
+                {title: 'Supporting Observations', prop: 'Description_ClinVar'}, // possibly Description_ClinVar
+                {title: 'Review Status', prop: 'Review_Status_ClinVar'},
+            ]
+        }
+    },
+    {groupTitle: 'Clinical Significance (LOVD)', internalGroupName: 'Significance (LOVD)', reportSource: 'LOVD',
+        reportBinding: {
+            submitter: {title: 'Submitter(s)', prop: 'Submitters_LOVD'},
+            cols: [
+                // displayed here in full since the display in the header is potentially truncated
+                {title: 'Submitter(s)', prop: 'Submitters_LOVD'},
+
+                // added this field (redundantly?) b/c it's not noticeable in the header
+                {title: 'Variant Effect', prop: 'Variant_effect_LOVD'},
+
+                {title: 'Genetic Origin', prop: 'Genetic_origin_LOVD'},
+                {title: 'Individuals', prop: 'Individuals_LOVD'},
+                {title: 'Submission ID', prop: 'DBID_LOVD'}, // possibly BX_ID_LOVD, DBID_LOVD?
```

Comment:
DBID is correct -- BX_ID's are brca_exchange ids for internal purposes only. can remove comment